twisted/python/_release.py

```diff
@@ -58,8 +58,8 @@ def runCommand(args, cwd=None):
     @raise CommandFailed: when the program exited with a non-0 exit code.
     """
     process = Popen(args, stdout=PIPE, stderr=STDOUT, cwd=cwd)
-    stdout = process.stdout.read()
-    exitCode = process.wait()
+    stdout = process.communicate()[0]
+    exitCode = process.poll()
```

Comment:
Ah OK, thanks for pointing that out Popen.communicate() waits for the child process to exit,
that's good.